{{cookiecutter.projectDirectory}}/{{cookiecutter.projectName}}/Code/AppDelegate.swift

```diff
@@ -17,6 +20,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         API.setup()
         CredentialsController.shared.resetOnNewInstallations()
 
+        setupForceUpdate()
```

Comment:
ich würde den check auch in `applicationWillEnterForeground(_:)` geben

Reply:
ist durch die SceneDelegate abgedeckt 👍